tests/Helpers/AccessControl.php

```diff
@@ -38,21 +38,35 @@
 
 use App\Models\User;
 use Illuminate\Support\Arr;
+use App\Settings\LicenseSettings;
 
 use function Pest\Laravel\actingAs;
 
 use AidingApp\Authorization\Enums\LicenseType;
 
-function testResourceRequiresPermissionForAccess(string $resource, string|array $permissions, string $method)
+function testResourceRequiresPermissionForAccess(string $resource, string|array $permissions, string $method, $featureProperty = null)
```

Comment:
How about calling this new argument `addons` instead and typing it to `?LicenseData`.

If it is not `null` then just set `$settings->data->addons` equal to it and save. Giving the dev the ability to configure all the addons at once and providing type safety.